crates/hir-expand/src/builtin_fn_macro.rs

```diff
@@ -770,3 +771,15 @@ fn option_env_expand(
 
     ExpandResult::ok(expanded)
 }
+
+fn quote_expand(
```

Comment:
This is a very interesting macro, it's a builtin macro that is actually backed by a proc-macro that rustc loads from the `proc_macro` library https://doc.rust-lang.org/proc_macro/fn.quote.html! Unsure how we can hook that up for now, but that might require this part to be nightly gated as well